help/destinations/catalog/advertising/tradedesk-emails-update

```diff
@@ -0,0 +1,138 @@
+| Export frequency | **[!UICONTROL Daily Batch]** | As a profile is updated in Experience Platform based on segment evaluation, the profile (identities) are updated once a day downstream to the destination platform. Read more about [batch uploads](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-types.html?lang=en#file-based).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+### Authenticate to Destination {#authenticate}
+
+[!DNL The Trade Desk] CRM Destination is a daily batch file upload and does not require authentication by the user.
+
+### Fill in Destination Details {#fill-in-details}
+
+Before you can send, or activate, audience data to a destination, you must set up a connection to your own destination platform. While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+
+*  **[!UICONTROL Account Type]**: Please choose the **[!UICONTROL Existing Account]** option. 
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future. 
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Advertiser ID]**: your [!DNL Trade Desk Advertiser ID], which can either be shared by your [!DNL Trade Desk] Account Manager or be found under [!DNL Advertiser Preferences] in the [!DNL Trade Desk] UI. 
+
+![Platform UI screenshot showing how to fill in destination details] (help/destinations/assets/catalog/advertising/tradedesk/configuredestination2.png)
+
+When connecting to the destination, setting a data governance policy is completely optional. Please review the Experience Platform [data governance overview](https://experienceleague.adobe.com/docs/experience-platform/data-governance/policies/overview.html?lang=en) for more details.  
+
+## Activate segments to this destination {#activate}
+
+See [activate audience data to batch profile export destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations.html?lang=en) for instructions on activating audience segments to a destination. 
+
+In the **[!UICONTROL Scheduling]** page, you can configure the schedule and the file names for each segment you are exporting. Configuring the schedule is mandatory, but configuring the file name is optional. 
+
+![Platform UI screenshot to schedule segment activation] (help/destinations/assets/catalog/advertising/tradedesk/schedulesegment1.png)
+
+>[!NOTE] 
+>
+>All segments activated to [!DNL The Trade Desk] CRM Destination are automatically set to a daily frequency and full file export.
+
+![Platform UI screenshot to schedule segment activation] (help/destinations/assets/catalog/advertising/tradedesk/schedulesegment2.png)
```

Comment:
please update the link to start with an `/`, otherwise it won't render. see example in line 89.